WalletWasabi.Fluent/Extensions/CurrencyExtensions.cs

```diff
@@ -34,7 +34,7 @@ public static Money CalculateDestinationAmount(this BuildTransactionResult resul
 
 	public static string FormattedBtc(this decimal amount)
 	{
-		return string.Format(FormatInfo, "{0:### ### ### ##0.#### ####}", amount).Trim();
+		return string.Format(FormatInfo, "{0:### ### ### ##0.########}", amount).Trim();
```

Comment:
```suggestion
		return string.Format(FormatInfo, "{0:### ### ### ##0.## ### ###}", amount).Trim();
```

Reply:
This introduces back the issue that his PR is trying to fix.

Reply:
I think the issue might be in different part of code base, I am looking into that again.

Reply:
I might be confused, is this for copy pasting the amount?

Reply:
It was but I found that it might not be right fix

Reply:
If it's about copy pasting, then I would suggest `#########.######## BTC`.

Other applications probably don't recognize the whitespace at all.

Reply:
It's about pasting and later editing, I have found other place that is causing the editing issue and trying to find proper fix right now

Reply:
I have reverted that change, not fix is using different solution as this was wrong.